### PR TITLE
Signup tests

### PR DESCRIPTION
**Pull request by HarryPerez**
## Trello Card
https://trello.com/c/DZXtCt8R/27-tests-y-documentaci%C3%B3n-de-sign-up

### CODE REVIEW

File: test/user.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      .request(server)
+      .post('/users')
+      .send({
+        firstName: 'Harry',
+        lastName: 'Potter',
+        email: 'franco.perez@wolox.com.ar',
+        password: '12345678'
+      })
+      .then(res => {
+        res.should.have.status(201);
```
Comment (molinajulian) [inline test/user.js:27]: You can check fields with db. (Remember you function `findBy`)

File: test/user.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      .request(server)
+      .post('/users')
+      .send({
+        lastName: 'wolox',
+        email: 'test@wolox.com.ar',
+        password: '1234567'
+      })
+      .then(res => {
+        res.should.have.status(422);
+        res.body.message[0].msg.should.equal('can not be empty');
```
Comment (molinajulian) [inline test/user.js:72]: What field can not be empty?

File: test/user.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+      .request(server)
+      .post('/users')
+      .send({
+        firstName: 'test',
+        lastName: 'wolox',
+        email: 'test@wolox.com.ar',
+        password: '12345678'
+      })
+      .then(res => {
+        res.should.have.status(500);
```
Comment (molinajulian) [inline test/user.js]: 500?

File: test/user.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
+const chai = require('chai'),
```
Comment (molinajulian) [inline test/user.js:1]: You can check test with `npm test` ?

File: test/user.js [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+      .post('/users')
+      .send({
+        firstName: 'test',
+        lastName: 'wolox',
+        email: 'test@wolox.com.ar',
+        password: '12345678'
+      })
+      .then(res => {
+        res.should.have.status(500);
+        res.body.message.should.equal('The user with email: test@wolox.com.ar already exists');
```
Comment (molinajulian) [inline test/user.js:43]: Why fail this test, if you is not create a another user with same email?

**Review by molinajulian — CHANGES_REQUESTED** [review]

File: test/user.js (hunk 4 above)
Comment (HarryPerez) [reply test/user.js:1]: Yes!

File: test/user.js (hunk 5 above)
Comment (HarryPerez) [reply test/user.js:43]: I use dataCreation.js to load some data to the db before each test.

**Review by molinajulian — APPROVED** [review]

**Pull request merged**